app-emulation/crun/crun-0.10.4.ebuild

```diff
@@ -0,0 +1,63 @@
+# Copyright 2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{5,6,7} )
+
+inherit eutils python-any-r1
+
+DESCRIPTION="A fast and low-memory footprint OCI Container Runtime fully written in C"
+HOMEPAGE="https://github.com/containers/crun"
+SRC_URI="https://github.com/containers/${PN}/releases/download/${PV}/${P}.tar.gz"
+
+LICENSE="GPL-3"
```

Comment:
https://github.com/containers/crun/blob/master/COPYING.libcrun
What's this? Could you verify the licenses altogether?